[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:50120/session {"capabilities": {"firstMatch": [{}], "alwaysMatch": {"browserName": "chrome", "platformName": "any", "goog:chromeOptions": {"extensions": [], "args": []}}}, "desiredCapabilities": {"browserName": "chrome", "version": "", "platform": "ANY", "goog:chromeOptions": {"extensions": [], "args": []}}}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:230 Starting new HTTP connection (1): 127.0.0.1:50120
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:442 http://127.0.0.1:50120 "POST /session HTTP/1.1" 200 721
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:50120/session/56062093d8c4c8d05d0c0c382d076be0/window/maximize {}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:442 http://127.0.0.1:50120 "POST /session/56062093d8c4c8d05d0c0c382d076be0/window/maximize HTTP/1.1" 200 51
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:50120/session/56062093d8c4c8d05d0c0c382d076be0/timeouts {"implicit": 30000}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:442 http://127.0.0.1:50120 "POST /session/56062093d8c4c8d05d0c0c382d076be0/timeouts HTTP/1.1" 200 14
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:50120/session/56062093d8c4c8d05d0c0c382d076be0/url {"url": "http://192.168.129.241:8288/#/home"}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:442 http://127.0.0.1:50120 "POST /session/56062093d8c4c8d05d0c0c382d076be0/url HTTP/1.1" 500 1215
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request